css/properties/display.json

```diff
@@ -52,91 +52,30 @@
         "contents": {
           "__compat": {
             "support": {
-              "chrome": [
-                {
-                  "version_added": "65"
-                },
-                {
-                  "version_added": "58",
-                  "version_removed": "65",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "Enable experimental Web Platform features"
-                    }
-                  ]
-                }
-              ],
-              "chrome_android": [
-                {
-                  "version_added": "65"
-                },
-                {
-                  "version_added": "58",
-                  "version_removed": "65",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "Enable experimental Web Platform features"
-                    }
-                  ]
-                }
-              ],
+              "chrome": {
+                "version_added": "65"
+              },
+              "chrome_android": {
+                "version_added": "65"
+              },
               "edge": {
                 "version_added": "79"
               },
-              "firefox": [
-                {
-                  "version_added": "37"
-                },
-                {
-                  "version_added": "36",
-                  "version_removed": "53",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "layout.css.display-contents.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                }
-              ],
+              "firefox": {
+                "version_added": "37"
+              },
               "firefox_android": {
                 "version_added": "57"
```

Comment:
Is the difference here real, or actually a bug in the data? It doesn't seem very plausible to me.

Reply:
The data originally came from #1271.  By the looks of it, the data was sourced from CanIUse, which doesn't retain data for Firefox Android other than the current version.  I doubt the data is correct, fixing!